app/src/main/java/org/ea/sqrl/activites/UrlLoginActivity.java

```diff
@@ -103,45 +103,7 @@ public void beforeTextChanged(CharSequence password, int start, int count, int a
             public void onTextChanged(CharSequence password, int start, int before, int count) {
                 if (!storage.hasQuickPass()) return;
                 if ((start + count) >= storage.getHintLength()) {
-                    showProgressPopup();
-                    closeKeyboard();
-
-                    new Thread(() -> {
-                        boolean decryptionOk = storage.decryptIdentityKey(password.toString(), entropyHarvester, true);
-                        if(!decryptionOk) {
-                            showErrorMessage(R.string.decrypt_identity_fail);
-                            handler.post(() -> {
-                                txtLoginPassword.setHint(R.string.login_identity_password);
-                                txtLoginPassword.setText("");
-                                hideProgressPopup();
-                            });
-                            storage.clear();
-                            storage.clearQuickPass();
-                            return;
-                        }
-
```

Comment:
In this code, if `decryptionOk`, there is no `showClearNotification()`, which the reason for the boolean.  Instance 1 of 2.

Reply:
One of the unfortunate inconsistencies :(

If you succeed in decrypting the identity we will set the quickpass and therefore we need to show the clear quickpass notification in the notification bar.